tests/helpers/default-packager.js

```diff
@@ -0,0 +1,275 @@
+};
+const DEFAULT_BOWER_COMPONENTS = {};
+
+const DEFAULT_NODE_MODULES = {
+  'broccoli-asset-rev': {
+    'index.js': 'module.exports = function() { return { name: "broccoli-asset-rev" }; };',
+    'package.json': JSON.stringify({
+      name: 'broccoli-asset-rev',
+      keywords: ['ember-addon'],
+    }),
+  },
+  'ember-cli-htmlbars': {
+    'index.js': `module.exports = {
+        name: 'ember-cli-htmlbars',
+        setupPreprocessorRegistry(type, registry) {
+          registry.add('template', {
+            name: 'ember-cli-htmlbars',
+            ext: 'hbs',
+            toTree(tree) { return tree; }
+          });
+        }
+      };
+      `,
+    'package.json': JSON.stringify({
+      name: 'ember-cli-htmlbars',
+      version: '2.0.1',
+      keywords: ['ember-addon'],
+    }),
+  },
+  'ember-source': {
+    'index.js': `module.exports = function() {
+        return {
+          name: 'ember-source',
+          paths: {
+            debug: 'vendor/ember/ember.debug.js',
+            prod: 'vendor/ember/ember.prod.js',
+            testing: 'vendor/ember/ember-testing.js',
+            shims: undefined,
+            jquery: 'vendor/ember/jquery/jquery.js',
+          }
+        };
+      };
+      `,
+    'package.json': JSON.stringify({
+      name: 'ember-source',
+      version: '~3.0.0-beta.1',
+      keywords: ['ember-addon'],
+      paths: {
+        debug: 'vendor/ember/ember.debug.js',
+        prod: 'vendor/ember/ember.prod.js',
+        testing: 'vendor/ember/ember-testing.js',
+        shims: undefined,
+        jquery: 'vendor/ember/jquery/jquery.js',
+      },
+    }),
+  },
+};
+
+const DEFAULT_VENDOR = {
+  'loader': {
+    'loader.js': 'W',
+  },
+  'ember': {
+    'jquery': {
+      'jquery.js': 'window.$ = function() {}',
+    },
+    'ember.js': 'window.Ember = {};',
+    'ember.debug.js': 'window.Ember = {};',
+  },
+  'ember-cli': {
+    'app-boot.js': '!!!',
+    'app-config.js': 'SPARTA',
+    'app-prefix.js': 'THIS',
+    'app-suffix.js': 'IS',
+    'test-support-prefix.js': 'If a clod be washed away by the sea,',
+    'test-support-suffix.js': 'Europe is the less.',
+    'tests-prefix.js': 'As well as if a promontory were.',
+    'tests-suffix.js': 'As well as if a manor of thine own',
+    'vendor-prefix.js': 'HELLO',
+    'vendor-suffix.js': '!',
+  },
+  'ember-cli-shims': {
+    'app-shims.js': 'L',
+  },
+  'ember-resolver': {
+    'legacy-shims.js': 'D',
+  },
+};
+
+const DEFAULT_SOURCE = {
+  'index.html': '<html></html>',
+  'router.js': 'export default class {}',
+  'resolver.js': `export default class {}`,
+  routes: {
+    'index.js': 'export default class {}',
+  },
+  styles: {
+    'app.css': 'html,body{height:100%}',
+  },
+  templates: {
+    'application.hbs': '{{outlet}}',
+  },
+  config: {
+    'environment.js': `module.exports = function() { return { modulePrefix: 'the-best-app-ever' }; };`,
+  },
+  'package.json': JSON.stringify({
+    name: 'the-best-app-ever',
```

Comment:
taken from other PR, does this need to be a complete package.json, or can we get by by only filling out necessary keys?

Reply:
Also, I just saw that this is copied code, so I'm not as concerned about you fixing.